From felipe.balbi@nokia.com  Mon Dec  7 16:43:15 2009
From: Felipe Balbi <felipe.balbi@nokia.com>
Date: Fri,  4 Dec 2009 15:47:45 +0200
Subject: USB: musb: omap2430: fix sparse warning
To: linux-usb@vger.kernel.org
Cc: Greg Kroah-Hartman <gregkh@suse.de>, Felipe Balbi <felipe.balbi@nokia.com>
Message-ID: <1259934467-14597-4-git-send-email-felipe.balbi@nokia.com>


Fix the following sparse warning:

drivers/usb/musb/omap2430.c:314:16: warning: Using plain integer as NULL pointer

Signed-off-by: Felipe Balbi <felipe.balbi@nokia.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/musb/omap2430.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/musb/omap2430.c
+++ b/drivers/usb/musb/omap2430.c
@@ -315,7 +315,7 @@ int musb_platform_exit(struct musb *musb
 	musb_platform_suspend(musb);
 
 	clk_put(musb->clock);
-	musb->clock = 0;
+	musb->clock = NULL;
 
 	return 0;
 }
